bin/readme-bld.sh

```diff
@@ -10,6 +10,7 @@ fi
 
 # tw5.com readmes
 node $TW5_BUILD_TIDDLYWIKI \
+	+plugins/tiddlywiki/multiwikiserver \
```

Comment:
Why does readme-bld.sh need the multiwikiserver plugin?

Reply:
It's a temporary measure while the PR is in development. It allows us to include content from the MWS plugin in the readme.